### ISSUE TITLE
failed to upgrade chart for release: function "lookup" not defined

### ISSUE THREAD

**Issue by nfickas** [fluxcd/helm-operator#335]
**Describe the bug**
When Flux attempts to upgrade my helm chart I get the error: `failed to upgrade chart for release: function "lookup" not defined`. I am currently using this line in my `deployment.yaml` file: 
```yaml
{{ (index ((lookup "v1" "Service" "default" "api").status.loadBalancer.ingress) 0).hostname }}
```

**To Reproduce**
Steps to reproduce the behaviour:
1. Install flux and the helm-operator via the helm charts.
2. Define a yaml file that uses the above template function to discover a loadbalancers url.
3. The error can be seen if you do a `k get helmrelease`.

**Expected behavior**

I expect the helm chart to deploy and have the loadbalancers uri put into an environment variable.

**Logs**
If applicable, please provide logs. In a standard stand-alone installation, you'd get this by running `kubectl logs -n default deploy/flux-helm-operator`.

**Additional context**
Add any other context about the problem here, e.g 
- Helm operator version: 1.0.0-rc9
- Targeted Helm version: v3.1.0
- Kubernetes version:
- Git provider:
- Container registry provider:

**Comment by ruheenaansari34** [comment]
Upgrading helm operator from 1.0.0-rc9 (uses patched helm 3.0.3 version) to 1.0.0 (uses Helm 3.1.2) resolves this issue. See https://github.com/bitnami/charts/issues/5091 for more details.

**Comment by kingdonb** [comment]
Sorry if your issue remains unresolved. The Helm Operator is in maintenance mode, we recommend everybody upgrades to Flux v2 and Helm Controller.

A new release of Helm Operator is out this week, 1.4.4.

We will continue to support Helm Operator in maintenance mode for an indefinite period of time, and eventually archive this repository.

Please be aware that Flux v2 has a vibrant and active developer community who are actively working through minor releases and delivering new features on the way to General Availability for Flux v2.

In the mean time, this repo will still be monitored, but support is basically limited to migration issues only. I will have to close many issues today without reading them all in detail because of time constraints. If your issue is very important, you are welcome to reopen it, but due to staleness of all issues at this point a new report is more likely to be in order. Please open another issue if you have unresolved problems that prevent your migration in the appropriate Flux v2 repo.

Helm Operator releases will continue as possible for a limited time, as a courtesy for those who still cannot migrate yet, but these are strongly not recommended for ongoing production use as our strict adherence to semver backward compatibility guarantees limit many dependencies and we can only upgrade them so far without breaking compatibility. So there are likely known CVEs that cannot be resolved.

We recommend upgrading to Flux v2 which is actively maintained ASAP.

I am going to go ahead and close every issue at once today,
Thanks for participating in Helm Operator and Flux! 💚 💙
